### PR TITLE
Blog Post: Long-Term Support For Dusty Daemon

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build Preview:**

- [/blog/dusty-daemon-receives-lts-grant](https://os-website-git-lts-dusty-daemon-open-sats.vercel.app/blog/dusty-daemon-receives-lts-grant)

**Social Preview:**

- image

### CODE REVIEW

File: data/blog/dusty-daemon-receives-lts-grant.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+and the design and implementation of Splicing.
+
+[Splicing](https://lightningsplice.com/splicing_explained.html) enhances the
+usability of the Lightning Network by allowing users to modify channel sizes
+instead of having to close and create new channels when size changes are
+required. This reduces the on-chain transactions required and improves the
+usability of the Lightning Network.
+
+With LTS support from OpenSats, Dusty will help extend the Splicing
+implementation across additional Lightning implementations and commence building
```
Comment (dergigi) [inline data/blog/dusty-daemon-receives-lts-grant.mdx:24]: Potentially re-word that part, as "Splicing implementation" followed by "Lightning implementations" sounds like a word repetition.

```suggestion
implementation across the Lightning ecosystem and commence building
```

**Comment by Arvin21M** [comment]
@OpenSats/volunteers Would appreciate a quick sanity check on this announcement when some one has a moment. 🙏

**Review by Bayernatoor — APPROVED** [review]
@Arvin21M Looks great to me. Reads well and all the links work :zap: :rocket:

**Comment by Arvin21M** [comment]
@dergigi We're good to go 🚀

**Review by btcschellingpt — APPROVED** [review]
Couldn't have written better myself! ;-)

**Review by dergigi — APPROVED** [review]
Looks good to me ✅

:shipit:

**Comment by dergigi** [comment]
Live now!
- https://opensats.org/blog/dusty-daemon-receives-lts-grant
- [nostr post](https://njump.me/nevent1qqsdhgsu9dkft5qc8hqsr2ezm6datkm3473n4e7yzpp5mxvg4usx36sppamhxue69uhkummnw3ezumt0d5q3gamnwvaz7tmjv4kxz7fwv3sk6atn9e5k7q3q0pensatlcfwktnvjjw2dtem38n6rvw8g6fv73h84cuacxn4c28eqjwlwz6)

**Pull request merged**